arch/src/x86_64/regs.rs

```diff
@@ -269,6 +269,11 @@ fn create_msr_entries() -> Vec<kvm_msr_entry> {
         data: msr_index::MSR_IA32_MISC_ENABLE_FAST_STRING as u64,
         ..Default::default()
     });
+    entries.push(kvm_msr_entry {
+        index: msr_index::MSR_MTRRdefType,
+        data: 0xc06,
```

Comment:
Could you write this with two constants: MTRR_ENABLE (= 0xc00)  | MTRR_WB ( =0x6)

Reply:
Done! Thanks for the suggestion :)